src/evo/specialtxman.h

```diff
@@ -5,10 +5,10 @@
 #ifndef BITCOIN_EVO_SPECIALTXMAN_H
 #define BITCOIN_EVO_SPECIALTXMAN_H
 
-#include <sync.h>
-#include <threadsafety.h>
 #include <gsl/pointers.h>
 #include <optional>
+#include <sync.h>
```

Comment:
nit: standard library headers should be at the bottom, `clang-tidy` doesn't tend to recognize this unless headers have an additional newline